src/index.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
- [ ] When I run the project, I was getting an error that said:  `Can't resolve 'react-router-dom'`, then, when I run `npm install react-router-dom --save` everything worked fine. Please add `react-router-dom`  to package.json by running `npm install react-router-dom --save`